docs.json

```diff
@@ -1,235 +1,238 @@
-							}
-						]
-					},
-					{
-						"group": "Transform",
-						"icon": "arrow-right-arrow-left",
-						"pages": [
-							{
-								"group": "Workflows",
-								"pages": [
-									"api-ref/transform/workflows/fetch-all-workflows",
-									"api-ref/transform/workflows/create-a-workflow",
-									"api-ref/transform/workflows/fetch-a-workflow",
-									"api-ref/transform/workflows/update-a-workflow"
-								]
-							},
-							{
-								"group": "Jobs",
-								"pages": [
-									"api-ref/transform/jobs/create-a-job-put",
-									"api-ref/transform/jobs/create-a-job-post",
-									"api-ref/transform/jobs/fetch-a-job",
-									"api-ref/transform/jobs/fetch-a-job-by-key"
-								]
-							}
-						]
-					},
-					{
-						"group": "Sequences",
-						"icon": "list",
-						"pages": [
-							"api-ref/sequences/introduction",
-							{
-								"group": "Series",
-								"pages": [
-									"api-ref/sequences/series/overview",
-									"api-ref/sequences/series/fetch-all-series",
-									"api-ref/sequences/series/create-a-series",
-									"api-ref/sequences/series/fetch-a-series"
-								]
-							},
-							{
-								"group": "Entries",
-								"pages": [
-									"api-ref/sequences/entries/overview",
-									"api-ref/sequences/entries/fetch-all-series-entries",
-									"api-ref/sequences/entries/create-a-series-entry",
-									"api-ref/sequences/entries/fetch-a-series-entry"
-								]
-							}
-						]
-					},
-					{
-						"group": "Access",
-						"icon": "building",
-						"pages": [
-							{
-								"group": "Workspaces",
-								"pages": [
-									"api-ref/access/workspace/fetch",
-									"api-ref/access/workspace/update"
-								]
-							},
-							{
-								"group": "Enrollments",
-								"pages": [
-									"api-ref/access/enrollment/authorize",
-									"api-ref/access/enrollment/fetch",
-									"api-ref/access/enrollment/update"
-								]
-							}
-						]
-					},
-					{
-						"group": "Apps",
-						"icon": "puzzle-piece",
-						"pages": [
-							{
-								"group": "PDF Generator",
-								"pages": ["api-ref/apps/pdf/preview"]
-							}
-						]
-					},
-					{
-						"group": "Utils",
-						"icon": "wrench",
-						"pages": ["api-ref/utils/ping", "api-ref/utils/uuid"]
-					}
-				]
-			}
-		]
-	},
-	"styling": {
-		"codeblocks": "system"
-	},
-	"logo": {
-		"light": "/logo/light.svg",
-		"dark": "/logo/dark.svg"
-	},
-	"api": {
-		"mdx": {
-			"server": "https://api.invopop.com",
-			"auth": {
-				"method": "bearer"
-			}
-		}
-	},
-	"navbar": {
-		"links": [
-			{
-				"label": "Contact Us",
-				"href": "https://www.invopop.com/contact"
-			}
-		]
-	},
-	"footer": {
-		"socials": {
-			"x": "https://x.com/goInvopop",
-			"linkedin": "https://www.linkedin.com/company/invopop/",
-			"github": "https://github.com/invopop"
-		}
-	},
-	"integrations": {
-		"plausible": {
-			"domain": "docs.invopop.com"
-		}
-	}
+  "$schema": "https://mintlify.com/docs.json",
```

Comment:
We need to be careful with indentation as this can cause no end of conflicts. If possible, please stick with tabs for the time being.